### PR TITLE
Fix invalid path on Windows when opening traces

### PR DESCRIPTION
**Pull request by hoangphamEclipse**
On Windows, Theia returns an invalid file path when opening traces. The invalid file path begins with a forward slash **/**. For example, the path **C:/** will become **/C:/**. This prevents the trace extension from opening trace files. This is a known Theia issue. More on the issue can be read [here](https://github.com/eclipse-theia/theia/issues/8098).

This commit resolves temporarily this issue by using [FileUri](https://eclipse-theia.github.io/theia/docs/next/modules/core.fileuri-1.html) to convert the original file path returned by Theia to a platform specific path. This will allow the trace extension to open traces on Windows. 

Signed-off-by: Hoang Thuan Pham &lt;hoang.pham@calian.ca&gt;

### CODE REVIEW

File: theia-extensions/viewer-prototype/src/node/backend-file-service-impl.ts [hunk 1]
```diff
@@ -4,17 +4,25 @@ import { Dirent } from 'fs';
 import { Path } from '@theia/core/lib/common/path';
 import { CancellationToken } from '@theia/core';
 import { BackendFileService } from '../common/backend-file-service';
+import { FileUri } from '@theia/core/lib/node';
 
 @injectable()
 export class BackendFileServiceImpl implements BackendFileService {
 
     async findTraces(path: string, cancellationToken: CancellationToken): Promise<string[]> {
+        /*
```
Comment (bhufmann) [inline theia-extensions/viewer-prototype/src/node/backend-file-service-impl.ts:13]: remove trailing whitespaces

**Review by bhufmann — CHANGES_REQUESTED** [review]
Your commit message needs to be formatted properly and you need to add a correctly formatted signed-off-by line

**Review by Rodrigoplp-work — APPROVED** [review]

**Review by bhufmann — CHANGES_REQUESTED** [review]
The commit message is not formatted properly has to long lines. The main readme has a section about how to contribute code and format the message.
https://github.com/theia-ide/theia-trace-extension#how-to-contribute-code

Please update the commit message accordingly.

**Review by bhufmann — APPROVED** [review]
Thanks for fixing this. Looks good to me.

**Review by Rodrigoplp-work — APPROVED** [review]

**Comment by bhufmann** [comment]
Fixes #545

**Pull request merged**